CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Models/GiniImageDocument.swift

```diff
@@ -86,14 +86,25 @@ final public class GiniImageDocument: NSObject, GiniCaptureDocument {
         guard let cgImage = image.cgImage else { return image }
         var updatedRect: CGRect
 
-        print("X: image.size.width \(image.size.width) image.size.height \(image.size.height)")
-
-
         if image.size.width < image.size.height {
-//            updatedRect = CGRect(x: 500, y: 20, width: 2000, height: 2800)
-            updatedRect = CGRect(x: 20, y: 500, width: 2800, height: 2000)
+            if UIDevice.current.isIpad {
+                updatedRect = CGRect(x: 350, y: 350, width: 2400, height: 3300)
+            } else {
+                updatedRect = CGRect(x: 20, y: 500, width: 2800, height: 2000)
+            }
         } else {
-            updatedRect = CGRect(x: 500, y: 20, width: 2000, height: 2800)
+            if UIDevice.current.isIpad {
+                if image.imageOrientation == .up {
+                    updatedRect = CGRect(x: 700, y: 50, width: 2100, height: 2900)
```

Comment:
[Suggestion] Could we extract these Rects in a separate func -> like calculateUpdatedRect() and do the calculation there with some inline comments 😊?

Reply:
Already added the comments to this functiono, but we can also separate it in a different function